lib/chore/unit_of_work.rb

```diff
@@ -12,10 +12,11 @@ module Chore
   class UnitOfWork < Struct.new(:id, :receipt_handle, :queue_name, :queue_timeout, :message, :previous_attempts, :consumer, :decoded_message, :klass)
     # The time at which this unit of work was created
     attr_accessor :created_at
-
+    attr_accessor :received_timestamp
     def initialize(*) #:nodoc:
       super
       @created_at = Time.now
+      @received_timestamp = Time.now
```

Comment:
For the **`Worker`** dedupe check
 The **`Worker`** version creates an explicit `received_timestamp` attr on `UnitOfWork`. It's the same value as created_at, but eliminates confusion when the attr is called.

Reply:
I did think about that, but decided to be more explicit since it's accessed as `received_timestamp` and _may_ cause confusion. I don't mind using either one though, it's a quick fix, just lmk!